pmm/v3/pmm3-release-candidate.groovy

```diff
@@ -42,6 +42,7 @@ void deleteReleaseBranches(String VERSION) {
 }
 
 void setupReleaseBranches(String VERSION) {
+    git branch: SUBMODULES_GIT_BRANCH, credentialsId: 'GitHub SSH Key', poll: false, url: 'git@github.com:Percona-Lab/pmm-submodules'
```

Comment:
I believe it's unnecessary if we look at https://github.com/Percona-Lab/jenkins-pipelines/pull/2611/files#diff-4df4cdc7b0a78d9c1d7ee808748e0a8c484be4ec2aedaf99df111c72c0a4e8c3R199

Reply:
By the time this function gets called, the repo is already cloned.